CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## 6.6.0
+## 6.6.1
 - Add recursive-related-bundles endpoint 
```

Comment:
Should we have two releases in the release notes? 6.6.1 only having the bug fix and the rest in 6.6.0

Reply:
Only 6.6.1 looks okay to me since 6.6.0 was not released, but no strong objections to having both releases here

Reply:
GitHub release of 6.6.0 was never done - we have only  v6.6.0 tag present...

Reply:
Ok, good to go with 6.6.1

Reply:
And I hope that v6.6.1 will be working I am planning to do the GitHub release - which will create official IIB images and release